### PR TITLE
Language changes

### PR DESCRIPTION
**Pull request by stavros-0**

### CODE REVIEW

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 1]
```diff
@@ -529,11 +534,16 @@ if: |
...
 question: |
+  % if person_answering == "tenant":
   Did your landlord ever sell the house while you were a tenant at this address?
+  % else:
+  Did the tenant's landlord ever sell the house while they were a tenant at this address?
+  % endif
+  
 fields:
-  - Was your house sold to a new landlord while you lived in it?: previous_landlord_sold_property
+  - ${"Was your house sold to a new landlord while you lived in it?" if person_answering == "tenant" else "Was the tenant's house sold to a new landlord while they lived in it?"}: previous_landlord_sold_property
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:544]:
```suggestion
  - label: |
      % if if person_answering == "tenant":
      Was your house sold to a new landlord while you lived in it?
      % else:
      Was the tenant's house sold to a new landlord while they lived in it?
      % endif
    field: previous_landlord_sold_property
```

Ternary/one-line if statements aren't so great for translation. We should avoid those.

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 2]
```diff
@@ -529,11 +534,16 @@ if: |
...
     datatype: yesnomaybe
-  - Did your new landlord give you a letter that told you about the sale?: notice_of_sale_provided_new_landlord
+  - ${"Did your new landlord give you a letter that told you about the sale?" if person_answering == "tenant" else "Did the tenant's new landlord give them a letter that told them about the sale?"}: notice_of_sale_provided_new_landlord
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:546]: Same comment as above--don't use one line if statements.

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 3]
```diff
@@ -674,8 +696,8 @@ fields:
     choices:
       - A lease for a fixed period of time (like 1 year): fixed_period
       - A month-to-month, week-to-week or other lease that automatically renews: periodic
-      - I don't have have a written lease: no_written_lease
-      - I don't know: dont_know
+      - ${ "I don't have have a written lease" if person_answering == "tenant" else "The tenant does not have a written lease"}: no_written_lease
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:699]:
```suggestion
      - label: |
          % if if person_answering == "tenant":
          I don't have have a written lease
          % else:
          The tenant does not have a written lease
          % endif
          value: no_written_lease
```

Or use the approach in UpToCode, where we have a lookup dictionary for options that we want to display in the review block later as well.

Same for the below option.

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 4]
```diff
@@ -1163,19 +1227,19 @@ fields:
       % endif
     field: got_notice_to_terminate
     datatype: yesnomaybe
-  - About what day did you **get** the notice?: received_notice_date
+  - ${"About what day did you **get** the notice?" if person_answering == "tenant" else "About what day did the tenant **get** the notice?"}: received_notice_date
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:1230]: See above notes, can't use ternary

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 5]
```diff
@@ -1163,19 +1227,19 @@ fields:
...
     datatype: date
     show if: got_notice_to_terminate
     max: ${ today().format("yyyy-MM-dd") }
-  - Did the notice include a specific day to **end** your tenancy?: notice_to_terminate_includes_specific_day
+  - ${"Did the notice include a specific day to **end** your tenancy?" if person_answering == "tenant" else "Did the notice include a specific day to **end** the tenant's tenancy?"}: notice_to_terminate_includes_specific_day
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:1234]: ternary

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 6]
```diff
@@ -1163,19 +1227,19 @@ fields:
...
     input type: radio
     choices:
       - Yes, it had a specific day: specific_day
       - It said to move at the end of the next rental period: rental_period
       - It did not say when to move: no_specific_day
       - It said something else: something_else
     show if: got_notice_to_terminate
-  - What day did the notice say it would end your tenancy?: termination_of_tenancy_date
+  - ${"What day did the notice say it would end your tenancy?" if person_answering == "tenant" else "What day did the notice say it would end the tenant's tenancy"}: termination_of_tenancy_date
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:1242]: ternary

File: docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml [hunk 7]
```diff
@@ -1619,14 +1693,22 @@ question: |
...
-
+  
+  % if person_answering == "tenant":
   If you are not sure, it is OK to leave this question blank.
+  % else:
+  If the tenant is not sure, it is OK to leave this question blank.
+  % endif
 fields:
-  - What law did your landlord violate?: estoppel_breaches
+  - ${"What law did your landlord violate?" if person_answering == "tenant" else "What law did the tenant's landlord violate?"}: estoppel_breaches
```
Comment (nonprofittechy) [inline docassemble/MOHUDEvictionProject/data/questions/defense_logic.yml:1711]: ternary statement

**Review by miabonardi — APPROVED** [review]

**Pull request merged**